app/lib/profile/models/hive/cpic_lookup_response.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:hive/hive.dart';
+
+part 'cpic_lookup_response.g.dart';
+
+typedef Lookup = Map<String, String>;
+
+@HiveType(typeId: 3)
+@JsonSerializable()
+class CpicLookup {
+  CpicLookup({
+    required this.genesymbol,
+    required this.diplotype,
+    required this.lookupkey,
+  });
+
+  factory CpicLookup.fromJson(Map<String, dynamic> json) =>
+      _$CpicLookupFromJson(json);
```

Comment:
```suggestion
  factory CpicLookup.fromJson(Map<String, dynamic> json) {
    return _$CpicLookupFromJson(json);
  }
```

Reply:
had to refactor the json decoder, resulting in the lambda being able to fit on one line. therefore, am keeping arrow notation